src/Microsoft.SqlTools.ServiceLayer/Agent/Contracts/AgentNotebookRequest.cs

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Agent.Contracts
+{
+    /// <summary>
```

Comment:
a couple formatting notes.  Please add a blank line between class definitions and in getters\setters please add a space character after `{` and before `}`.  Take a look at some of the other class for examples.